cycles/testdata/cycle-within-pkg/conjure/com/palantir/services/services.conjure.go

```diff
@@ -35,7 +34,7 @@ func (c *myServiceClient) Endpoint1(ctx context.Context, authHeader bearertoken.
 	requestParams = append(requestParams, httpclient.WithRPCMethodName("Endpoint1"))
 	requestParams = append(requestParams, httpclient.WithRequestMethod("GET"))
 	requestParams = append(requestParams, httpclient.WithHeader("Authorization", fmt.Sprint("Bearer ", authHeader)))
-	requestParams = append(requestParams, httpclient.WithPathf("/endpoint1", url.PathEscape(fmt.Sprint(arg1Arg))))
+	requestParams = append(requestParams, httpclient.WithPathf("/endpoint1"))
```

Comment:
this is an unintentional fix. Previously this was an invalid printf statement and now it has the correct number of pattern values (none).